### PR TITLE
[tflite] Fix for CalibratorTest

### PR DESCRIPTION
**Pull request by akarmi**
CalibratorTest.test_invalid_model_buffer and
CalibratorTest.test_invalid_shape_calibrator_gen fail when called using the test target:
//tensorflow/lite/python/... and python3.

Log error message for test_invalid_model_buffer test:

> ======================================================================
> ERROR: test_invalid_model_buffer (__main__.CalibratorTest)
> test_invalid_model_buffer (__main__.CalibratorTest)
> ----------------------------------------------------------------------
> Traceback (most recent call last):
>   File "/home/antkac01/.cache/bazel/_bazel_antkac01/3a4b9f64af38ced70a092937bf072e49/execroot/org_tensorflow/bazel-out/k8-py2-opt/bin/tensorflow/lite/python/optimize/calibrator_test.runfiles/org_tensorflow/tensorflow/lite/python/optimize/calibrator_test.py", line 83, in test_invalid_model_buffer
>     with self.assertRaisesWithRegexpMatch(ValueError,
> AttributeError: 'CalibratorTest' object has no attribute 'assertRaisesWithRegexpMatch'
> 

The function assertRaisesRegex is provided by unittest module and it looks that it should be used here.
The old function is not found in the class CalibratorTest.

### CODE REVIEW

File: tensorflow/lite/python/optimize/calibrator_test.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def input_gen():
 
   def test_invalid_model_buffer(self):
     float_model = b'\0' * 100
-    with self.assertRaisesWithRegexpMatch(ValueError,
+    with self.assertRaisesRegex(ValueError,
                                           'Failed to parse the model'):
```
Comment (suharshs) [inline tensorflow/lite/python/optimize/calibrator_test.py]: thanks could you also fix the formatting here, now that the indent has changed.

**Review by suharshs — CHANGES_REQUESTED** [review]

**Comment by gbaned** [comment]
@akarmi Could you please check reviewer comments and keep us posted. Thanks!

File: tensorflow/lite/python/optimize/calibrator_test.py (hunk 1 above)
Comment (akarmi) [reply tensorflow/lite/python/optimize/calibrator_test.py]: Thanks for the feedback. Done.

**Comment by akarmi** [comment]
@gbaned We have addressed the reviewer's comments. Thanks.

**Comment by gbaned** [comment]
@akarmi  Could you please resolve the conflicts? Thanks!

**Comment by akarmi** [comment]
> 
> 
> @akarmi Could you please resolve the conflicts? Thanks!

@gbaned, done, ready to merge.

**Review by jianlijianli — APPROVED** [review]
Thanks!

**Pull request merged**